README.md

```diff
@@ -180,17 +182,59 @@ Here are quick questions to help you determine which formatting to use, dependin
 |----------------|--------------------------------------|
 | Indicate the structure of the content | Blocks        |
 | Enter additional data/metadata        | Entities      |
-| Format a portion of the text          | Inline styles |
+| Format a portion of a line            | Inline styles |
 
-#### Blocks
+#### Custom blocks
 
 Simple blocks are very easy to create. Add a new block type to `blockTypes`, specifying which `element` and `className` to display the block as.
 
+Here is an example, creating a "Terms & conditions" block:
+
+```jsx
+blockTypes={[
+    {
+        label: 'T&C',
+        type: 'terms-and-conditions',
+        element: 'div',
+        className: 'u-smalltext',
+    },
+]}
+```
+
 More advanced blocks, requiring custom React components, aren't supported at the moment. If you need this, feel free to [create an issue](https://github.com/springload/draftail/issues/new).
 
 #### Custom inline styles
 
-Custom inline styles aren't supported at the moment. This is on the roadmap, please refer to [#36](https://github.com/springload/draftail/issues/36).
+Custom inline styles only require a `style` prop, defining which CSS properties to apply when the format is active.
```

Comment:
It's not actually required – but if it's not there, there will be no way to tell whether the styles are active or not.